chimney/src/main/scala/io/scalaland/chimney/PartialTransformer.scala

```diff
@@ -58,6 +58,33 @@ trait PartialTransformer[From, To] extends PartialTransformer.AutoDerived[From,
     */
   final def transformFailFast(src: From): partial.Result[To] =
     transform(src, failFast = true)
+
+  /** Creates a new [[io.scalaland.chimney.PartialTransformer PartialTransformer]] by applying a pure function to a
+    * [[io.scalaland.chimney.partial.Result Result]] of transforming `From` to `To`. See an example:
+    * {{{
+    *   val stringTransformer: PartialTransformer[String, Int] =
+    *     PartialTransformer.fromFunction(_.length)
+    *
+    *   case class Length(length: Int)
+    *
+    *   implicit val toLengthTransformer: PartialTransformer[String, Length] =
+    *     stringTransformer.map(id => Length(id))
+    * }}}
+    *
+    * @param f
+    *   a pure function that maps a value of `To` to `A`
+    * @return
+    *   new [[io.scalaland.chimney.PartialTransformer PartialTransformer]]
+    *
+    * @since 1.5.0
+    */
+  final def map[A](f: To => A): PartialTransformer[From, A] = new PartialTransformer[From, A] {
```

Comment:
If there is `map` in partial, there could also be `mapPartial[A](f: To => partial.Result[A]): PartialTransformer[From, A]`.

Since `Transformer` has `contramap` the `PartialTransformer` should also get one.

Reply:
I thought about this once again and don't see any issue with adding `contramap` to `PartialTransformer` and `map` to `Transformer`, respectively.

What about `mapPartial`, it might be useful indeed, but don't you mind if we would add it separately?